Gemfile.next.lock

```diff
@@ -0,0 +1,435 @@
+      jaro_winkler (~> 1.5.1)
+      parallel (~> 1.10)
+      parser (>= 2.7.0.1)
+      rainbow (>= 2.2.2, < 4.0)
+      rexml
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (>= 1.4.0, < 2.0)
+    ruby-progressbar (1.10.1)
+    sass (3.7.2)
+      sass-listen (~> 4.0.0)
+    sass-listen (4.0.0)
+      rb-fsevent (~> 0.9, >= 0.9.4)
+      rb-inotify (~> 0.9, >= 0.9.7)
+    sass-rails (5.0.7)
+      railties (>= 4.0.0, < 6)
+      sass (~> 3.1)
+      sprockets (>= 2.8, < 4.0)
+      sprockets-rails (>= 2.0, < 4.0)
+      tilt (>= 1.1, < 3)
+    sassc (2.0.0)
+      ffi (~> 1.9.6)
+      rake
+    sentry-raven (2.13.0)
+      faraday (>= 0.7.6, < 1.0)
+    sidekiq (5.2.10)
+      connection_pool (~> 2.2, >= 2.2.2)
+      rack (~> 2.0)
+      rack-protection (>= 1.5.0)
+      redis (~> 4.5, < 4.6.0)
+    sidekiq-congestion (0.1.1)
+      congestion (~> 0.1)
+      sidekiq (>= 3.0)
+    sidekiq-logstash (1.2.0)
+      logstash-event (~> 1.2)
+      sidekiq (>= 3.0, < 6)
+    sidekiq-unique-jobs (7.1.5)
+      brpoplpush-redis_script (> 0.1.1, <= 2.0.0)
+      concurrent-ruby (~> 1.0, >= 1.0.5)
+      sidekiq (>= 5.0, < 7.0)
+      thor (>= 0.20, < 2.0)
+    simple_form (5.0.0)
+      actionpack (>= 5.0)
+      activemodel (>= 5.0)
+    spring (2.1.0)
+    spring-commands-rspec (1.0.4)
+      spring (>= 0.9.1)
+    spring-watcher-listen (2.0.1)
+      listen (>= 2.7, < 4.0)
+      spring (>= 1.2, < 3.0)
+    sprockets (3.7.2)
+      concurrent-ruby (~> 1.0)
+      rack (> 1, < 3)
+    sprockets-rails (3.4.2)
+      actionpack (>= 5.2)
+      activesupport (>= 5.2)
+      sprockets (>= 3.0.0)
+    stoplight (2.2.1)
+    strong_migrations (0.4.1)
+      activerecord (>= 5)
+    ten_years_rails (1.0.2)
+      actionview (~> 5.2.3)
+      rainbow (~> 3.0.0)
+    thor (1.2.1)
+    thread_safe (0.3.6)
+    tilt (2.0.8)
+    to_regexp (0.2.1)
+    tzinfo (1.2.10)
+      thread_safe (~> 0.1)
+    uglifier (4.2.0)
+      execjs (>= 0.3.0, < 3)
+    unf (0.1.4)
+      unf_ext
+    unf_ext (0.0.8.2)
+    unicode-display_width (1.7.0)
+    web-console (3.7.0)
+      actionview (>= 5.0)
+      activemodel (>= 5.0)
+      bindex (>= 0.4.0)
+      railties (>= 5.0)
+    webmock (3.14.0)
+      addressable (>= 2.8.0)
+      crack (>= 0.3.2)
+      hashdiff (>= 0.4.0, < 2.0.0)
+    websocket-driver (0.7.5)
+      websocket-extensions (>= 0.1.0)
+    websocket-extensions (0.1.5)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  active_record_extended
+  aws-sdk-s3
+  aws-sdk-sqs
+  bootsnap (>= 1.1.0)
+  bootstrap-sass (~> 3.4.1)
+  byebug
+  deferred_associations
+  factory_bot_rails
+  flipper
+  flipper-active_record
+  flipper-ui
+  graphiql-rails
+  graphql
+  httparty
+  jquery-rails
+  jsonpath
+  listen (>= 3.0.5, < 3.8)
+  lograge
+  logstash-event
+  newrelic_rpm
+  omniauth
+  omniauth-zooniverse
+  panoptes-client (~> 1.2)
+  pg (~> 1.3)
+  pry-byebug
+  pry-rails
+  puma (~> 5.6)
+  pundit (~> 2.2.0)
+  rack-cors
+  rails (~> 5.2)
```

Comment:
Hey Toyosi!

Apologies I did not notice this before. But something looks off with -next.lock. Your .next.lock gemfile should run on 6.0 but looks like its still on 5.2